### ISSUE TITLE
No process is associated with this object

### ISSUE THREAD

**Issue by Felipierucci** [rocksdanister/lively#2040]
When I try to put any wallpaper, this message appears: "No process is associated with this object. Exception: exception". It doesn't matter if it's one of the includes on the app or any external wallpaper  

![image](https://github.com/rocksdanister/lively/assets/153476954/1e0e981d-1677-4e67-92da-6f0246874cc0)

**Desktop (please complete the following information):**
Windows pro 22H2.

[lively_log_20231211_002335.zip](https://github.com/rocksdanister/lively/files/13630701/lively_log_20231211_002335.zip)

**Comment by rocksdanister** [comment]
Try the following:
1. https://github.com/rocksdanister/lively/wiki/Common-Problems#web-wallpapers-not-working
2. Check if any of the Lively files is deleted by your Anti-virus program if any.

**Comment by Felipierucci** [comment]
Hi! I don't have any anti-virus. I tried the solution and didn't worked.

**Comment by PIKACHU-977** [comment]
I have the same problem.
I have 2 monitors and at first when I turn on the computer the 2 backgrounds are applied.
But after a while the background of my main monitor is removed and I get the error "No process is associated with this object. Exception: exception" if I try to apply any background.

[lively_log_20231222_161350.zip](https://github.com/rocksdanister/lively/files/13753860/lively_log_20231222_161350.zip)

**Comment by nevesiya** [comment]
Hello! I have such a mistake too. 
When you turn on the PC, the application starts automatically, but the background is not applied, and when you manually try to install the phone, this error appears, I attach the logs.

[lively_log_20231226_181609.zip](https://github.com/rocksdanister/lively/files/13772593/lively_log_20231226_181609.zip)

**Comment by rocksdanister** [comment]
This could be MPV having issue with audio, you can try disabling: https://github.com/rocksdanister/lively/issues/1514#issuecomment-1368348842

**Comment by nevesiya** [comment]
Thanks! At the moment, it seems to have worked, I do not observe any errors.
If something changes, I'll let you know.

**Comment by nevesiya** [comment]
Unfortunately, I ran into a problem again, the same error. 
I am attaching a screenshot from the developer console and logs.
![2023-12-28 171350](https://github.com/rocksdanister/lively/assets/81850783/bdabef89-ddd6-47f9-a269-694150eea7bc)
[lively_log_20231228_171322.zip](https://github.com/rocksdanister/lively/files/13787624/lively_log_20231228_171322.zip)

**Comment by rocksdanister** [comment]
Things to try:
1. Disabling GPU Decode (Settings -> Wallpaper -> Video Player -> GPU Decode -> OFF)
2. (Only for installer version of Lively) Trying the latest MPV build from [here.](https://sourceforge.net/projects/mpv-player-windows/files/32bit/)
Extract and copy to Install `location/Plugins/Mpv`

**Comment by PIKACHU-977** [comment]
I tried to disable the GPU decoder but it still doesn't work, I get this error

<img width="883" alt="Captura de pantalla 2023-12-29 213445" src="https://github.com/rocksdanister/lively/assets/154615087/e691d5a1-38d7-4c24-b15e-64ecf3c30b28">

**Comment by moortezaa** [comment]
Well Hello There
I have the same issue here is the log

[lively_log_20240222_143825.zip](https://github.com/rocksdanister/lively/files/14373097/lively_log_20240222_143825.zip)

when I restart the app it works with these wallpapers:

1. Triangles & Light
2. Matrix Rain
3. Simple System
4. The Hill
5. Protein Clouds
6. Rain
7. Parallax
8. Medusae
9. Periodic Table

And These Ones Make The Problem

1. Fluids
2. Living Room
3. Music TV

And When The Problem Starts All of the wallpapers give the same Error
If I Use the tray icon and select close wallpapers I can run the fine ones
but the problematics are still going.

I should note that the Fluids wallpaper was working just fine A while ago
And I didn't do an update or any changes.
Just switched to the Triangles & Light and switched back to see the Fluids don't work anymore

**Comment by makee12** [comment]
I managed to get it to work by unchecking the 'Animate controls and elements inside windows". Now its working perfectly.
![image](https://github.com/user-attachments/assets/f35798fe-4f5a-4a22-8f0c-a535742a21b3)

**Comment by rocksdanister** [comment]
Are you on Windows 24H2 ?

Then windows icons need to be enabled: rightclick desktop -> views -> show desktop icon.

I think disabling animation is a temporary fix.

**Comment by Ralaphant** [comment]
Hi I've just had this issue after a unexpected shutdown of my cp, tried all of the fixes above and none are seeming to do anything. if i close lively and then reopen the wallpapers will appear with their motions but then freeze and shut down, well just turn off even though it is still registering it as an applied wallpaper. A fix soon would be great 

![Image](https://github.com/user-attachments/assets/35bac094-6e50-426c-8c1f-15ba289e0671)

**Comment by Ralaphant** [comment]
Also just to mention I get the error message once i open lively and try to select any other wallpaper 

![Image](https://github.com/user-attachments/assets/4f156a5c-d69d-4d06-99bd-f5373b1334a2)

**Comment by K0nt3xt** [comment]
I'm currently having the same issue. But i do notice it only happen when i use Roon Music Player. Seem like Roon audio transport thing conflicted with the newly used window MPV ? 

![Image](https://github.com/user-attachments/assets/94fbe63b-3be8-4c9f-9a52-9cf71e684e9a)

![Image](https://github.com/user-attachments/assets/6e4bab18-4e93-487c-acc4-2fed4d2088b5)

**Comment by BsWithBs** [comment]
I'm having the same issue here the wallpaper works for a while and then stops but fiddling around with the placement method fixes the error for me.

**Comment by Aquaticat** [comment]
I'm having the same issue trying to set Tabby terminal as the wallpaper for multiple monitors.
Log: 

[lively_log_20250512_132718.zip](https://github.com/user-attachments/files/20166912/lively_log_20250512_132718.zip)

**Comment by CGRoffcialcode** [comment]
[20250527_131144.txt](https://github.com/user-attachments/files/20459732/20250527_131144.txt)

Me too @Aquaticat I'm using windows terminal, trying to run an executable with it.
Problem is, I think the wallpapers are running, its just not displaying because the terminal runs when I try use the wallpaper, but it doesn't display as a wallpaper, along with the error
I've tried disabling audio for mpv but still gets error

![Image](https://github.com/user-attachments/assets/2c93861c-3927-457a-92e2-e4828344339c)
would like a fix soon

Edit:
With some debugging, I think for my end of the isuse I have some idea of what's happened.
Lively runs the terminal, expecting wt to appear, but with the argument that had been passed on, it changes the window to a different one, making it not possible for lively to trail it, giving a fall back error.

**Comment by Tono84** [comment]
I have been having this issue for a while. I was getting the same error message that everybody else was getting. Also when I selected the wallpaper, you could easily see that the PC was getting slow and the background changed every few secs between the livewallpaper and the reguglar background by default. 

I tried different things. I used different video player and did not work. I unchecked the "animate controls and elements..." option under performance options and the wallpaper started working.  It was showing, you could see that it was lagging a bit but tecnically was working. Seeing the messages above I believe I can find a fix. I will work on it on my free time. I will share any updates too. Hopefully we get this one working in no time.

**Comment by CGRoffcialcode** [comment]
@Tono84 
Look into the source code, I believe somewhere in the core folder, the error occurred there

**Comment by rocksdanister** [comment]
No process error means the program just crashed on startup.

You can't launch the WindowsApp folder apps like that I believe since it's sandboxed.

**Comment by CGRoffcialcode** [comment]
@rocksdanister yes, but somehow it still ran the cli executable, it didnt crash on startup, but Lively crashed on the error

**Comment by dervarex** [comment]
I have the same Issue, and sometimes Lively Wallpapers also crashes

[lively_log_20250713_162352.zip](https://github.com/user-attachments/files/21204794/lively_log_20250713_162352.zip)

Yesterday it worked

No process is associated with this object.
Exception: 
Exception

**Comment by Tono84** [comment]
Quick question. I was looking into this issue and while playing around with it, I noticed that I get this error when I have a slideshow running on my background, configured in windows. If I set a single photo as a backgraund while using livelwallpaper, it works without issues. Has anyone else seen this happening too with slideshows or are you guys getting the error with single photo?

**Comment by rocksdanister** [comment]
Some of these bug reports including the slideshow one is related to Windows changes things in the new update, this has been patched and to be pushed next update.
